lib/web_ui/lib/src/engine/canvaskit/image_web_codecs.dart

```diff
@@ -0,0 +1,436 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
2021?

Reply:
Our project uses 2013 everywhere. Using a future date is problematic, but using an older date is fine.